pybind/wallet3/wallet.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../common.hpp"
+
+#include <wallet3/wallet.hpp>
+#include <wallet3/default_daemon_comms.hpp>
+#include <wallet3/keyring.hpp>
+#include <wallet3/config/config.hpp>
+#include <oxenmq/oxenmq.h>
+
+namespace wallet3
```

Comment:
I'd put this in the `wallet` namespace instead of wallet3, to match what wallet3 core itself is doing, but also so that you can drop a ton of `wallet::` namespace prefixes in the code below.